core/src/avm2/globals/flash/events/IOErrorEvent.as

```diff
@@ -1,7 +1,23 @@
 package flash.events {
     public class IOErrorEvent extends ErrorEvent {
```

Comment:
https://ruffle.rs/compatibility/avm2 also has `DISK_ERROR`, `NETWORK_ERROR` and `VERIFY_ERROR`, but they are not listed in Flash documentation.

Reply:
Huh, those are actually the reason I touched this class. Just didn't notice them missing from the docs... :sweat_smile:

Reply:
Well they are supposed to be `[Inspectable(environment="none")]` - whatever that means.
But I'm not seeing any way for us to denote or enforce that...

Reply:
I have added them. Thanks!

Reply:
I'm pretty sure `[Inspectable(environment="none")]` is metadata that the Flash IDE uses